src/Microsoft.DotNet.Migration/DefaultMigrationRuleSet.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+using System.Linq;
+using System.IO;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Migration.Rules;
+
+namespace Microsoft.DotNet.Migration
```

Comment:
I was thinking this as well. `ProjectModel.Migration` or `ProjectJson.Migration` both seem pretty good.